common/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
-    package="com.microsoft.identity.common">
+    xmlns:tools="http://schemas.android.com/tools">
```

Comment:
I did not find this directly in the documentation you linked in the description.

Reply:
This is the documentation https://developer.android.com/build/publish-library/prep-lib-release#choose-namespace
Updated description